tests/unit/AddEditOrgPerson.spec.ts

```diff
@@ -319,7 +319,7 @@ describe('Add/Edit Org/Person component', () => {
 
     const messages = wrapper.findAll('.v-messages__message')
     expect(messages.length).toBe(3)
-    expect(messages.at(0).text()).toBe('Cannot exceed 30 characters')
+    expect(messages.at(0).text()).toBe('Cannot exceed 20 characters')
     expect(messages.at(1).text()).toBe('Cannot exceed 30 characters')
     expect(messages.at(2).text()).toBe('Cannot exceed 30 characters')
```

Comment:
@argush3 Should we be updating the middle name and last name lengths too?

Reply:
we are good on last name as it is 30 in COLIN.  I'm just confirming with @rarmitag if he ever sees middle name.  It could potentially run into the same issue though.

Reply:
@lewischenstudio lets update the middle name field to 20 in the schema and the UIs as it's possible that we get the same scenario as first name

Reply:
Sure.